test/src/main/scala/scala/test/twitter_scrooge/BUILD

```diff
@@ -18,6 +24,7 @@ scrooge_scala_library(
     "//test/src/main/scala/scala/test/twitter_scrooge/thrift/thrift2:thrift2_a",
     ":scrooge3",
   ],
+  exports = [":scrooge3"],
```

Comment:
why are all the scrooge deps also exported now? Can you add a comment next to each export explaining why it is needed?